torch/ao/quantization/fuser_method_mappings.py

```diff
@@ -235,6 +267,11 @@ def get_fuser_method_new(
     if fuser_method_mapping is None:
         fuser_method_mapping = DEFAULT_PATTERN_TO_FUSER_METHOD
 
-    fuser_method = fuser_method_mapping.get(op_pattern, None)
+    op_patterns = get_valid_patterns(op_pattern)
```

Comment:
could we clarify the naming?  `op_patterns = some_function(op_pattern)` is confusing because the name of the result is only one character away from the name of the input variable

Reply:
yeah sure